pkg/controller/common/version/version.go

```diff
@@ -25,7 +25,7 @@ var GlobalMinStackVersion Version
 var (
 	SupportedAPMServerVersions        = MinMaxVersion{Min: From(6, 2, 0), Max: From(9, 99, 99)}
 	SupportedEnterpriseSearchVersions = MinMaxVersion{Min: From(7, 7, 0), Max: From(8, 99, 99)}
```

Comment:
If `7.17.x` is the new min. supported version should we also update all of this?
```suggestion
	SupportedAPMServerVersions        = MinMaxVersion{Min: From(7, 17, 0), Max: From(9, 99, 99)}
	SupportedEnterpriseSearchVersions = MinMaxVersion{Min: From(7, 17, 0), Max: From(8, 99, 99)}
```

Reply:
I think this is expected given that we have gradually added support for them over time.